portfolio.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+ <link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Readex+Pro:wght@200&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="css/portfolio.css">
+  <title>Program Portfolio</title>
+</head>
+<body>
+  <header>
+   <center><h1>Programming Portfolio</h1></center>
+  </header>
+
+  <div>
+    <div class="grid-container"> 
+      <div class="grid-spacing"> <a href="index.html">Home</a></div>
+      <div class="grid-spacing"> <a href="about.html">About</a></div>
+      <div class="grid-spacing"> <a href="portfolio.html">Portfolio</a></div>
+    </div>
+  </div>
+<div class="flex-container">
+  <h4> Knowledgable In:</h4>
+<ul>
+  <li>🐍 Python</li>
+  <li>🏢 Building APIs</li>
+  <li>🔳 SQL</li>
+  <li>✔️ HTML</li>
+  <li>👁‍🗨 CSS</li>
+  <li>➿ React</li>
+</ul>
+</div>
+
+
+<h4>Projects:</h4>
+<ol>
+  <li><a href="https://github.com/Sanderspat/snowbug"> ☃️ ❄️ SNOWBUG  </a></li>
+  <li><a href= "https://github.com/Sanderspat/task-list-api"> ✅ Task List API</a></li>
+  <li><a href=" https://github.com/shainabeth/solar-system-api"> Pair Programming activity - 🌌 Solar System API</a></li>
+  <li><a href="https://github.com/Sanderspat/swap-meet"> 🛒 Swap Meet </a></li>
+  <li><a href= "https://github.com/Sanderspat/retro-video-store"> 📼 Retro Video Store </a></li>
+</ol>
+<footer id="footer">
+  <p>
+    ✨Thank you for visiting✨
+  </p>
+</body>
```

Comment:
Missing the closing footer tag here. A validator can help us catch issues like this.